SchemaInventory.json

```diff
@@ -349,6 +349,20 @@
       "date": "1/19/2023",
       "dynamic": "No",
       "approved": "Yes"
+    },
+    {
+      "name": "ECDbMap",
+      "path": "Standard\\ECDb\\Released\\ECDbMap.02.00.02.ecschema.xml",
+      "released": true,
+      "version": "02.00.02",
+      "comment": "",
+      "verifier": "",
```

Comment:
Just add a verifier here. Otherwise the PR looks good to me.

Reply:
done